incubator/group/keeper.go

```diff
@@ -35,8 +38,19 @@ const (
 	VoteByVoterIndexPrefix        byte = 0x42
 )
 
+type ProposalI interface {
+	GetBase() ProposalBase
+	SetBase(ProposalBase)
+}
+
+type ProposalModel interface {
+	orm.Persistent
+	GetProposalI() ProposalI
+}
+
 type Keeper struct {
-	key sdk.StoreKey
+	key               sdk.StoreKey
+	proposalModelType reflect.Type
```

Comment:
```suggestion
	proposalIType reflect.Type
```

Reply:
I don't mind renaming the type. I found the `I` suffix for interfaces very uncommon in Go projects that's why I choose Model instead. But that does not describe anything better.